frontend/app/user/people-list.component.js

```diff
@@ -32,6 +32,20 @@ export default class PeopleList extends React.Component {
     });
   };
 
+  deleteCase = (person, kase, e) => {
+    e.preventDefault();
+    const array = [...this.state.cases];
+    remove(array, item => {
+      return item === kase;
+    });
+    this.setState({ cases: array });
+    context.setContext({
+      activePerson: "",
```

Comment:
Why do we want to reset the activePerson when a case is deleted?

Reply:
I suppose only `activeCase` can be reset, because in `frontend/app/user/dashboard.component.js`, only when both `activeCase` and `activePerson` are truthy it displays the info:
```
{context.activePerson && context.activeCase && (
  <p>
    You are currently working on case{" "}
    <strong>{context.activeCase.name}</strong> for{" "}
    <strong>{context.activePerson.name}</strong>.
  </p>
)}
```